cognito-pre-token-generation/src/main/java/no/unit/nva/cognito/service/UserPoolEntryUpdater.java

```diff
@@ -64,6 +67,26 @@ private static AWSCognitoIdentityProvider defaultCognitoProvider() {
             .build();
     }
 
+    private void verifyThatCognitoHasUpdatedEntries(UserDetails userDetails, List<AttributeType> desiredAttributes) {
+        AdminGetUserResult response = getCognitoUserDetails(userDetails);
+        List<AttributeType> actualAttributes = response.getUserAttributes();
+        desiredAttributes.removeAll(actualAttributes);
+        if (!desiredAttributes.isEmpty()) {
+            logger.warn(COGNITO_UPDATE_FAILURE_WARNING + toString(desiredAttributes));
```

Comment:
Validation only for logging?

Reply:
I am only interested in logging at this point. I have not decided what to do next. I will decide after I see the logs.